elide-graphql/src/main/java/com/yahoo/elide/graphql/parser/GraphQLEntityProjectionMaker.java

```diff
@@ -568,13 +554,24 @@ private void addAttributeArgument(Argument argument, EntityProjectionBuilder pro
         String argumentName = argument.getName();
         Class<?> entityType = projectionBuilder.getType();
 
-        Attribute argumentAttribute = projectionBuilder.getAttributeByAlias(argumentName);
+        Attribute existingAttribute = projectionBuilder.getAttributeByAlias(argumentName);
+
         com.yahoo.elide.request.Argument elideArgument = com.yahoo.elide.request.Argument.builder()
                 .name(argumentName)
                 .value(variableResolver.resolveValue(argument.getValue()))
                 .build();
 
-        if (argumentAttribute == null) {
+        if (existingAttribute != null) {
```

Comment:
Do we really need to differentiate these two cases.  The resulting code in each branch is nearly identical.